packages/cli/vitest.setup.ts

```diff
@@ -2,6 +2,7 @@ import { mockServer } from "./mocks/server";
 
 const ALLOW_REGEX_LIST = [
   /https:\/\/api\.github\.com\/repos/,
+  /https:\/\/raw\.githubusercontent\.com\/wrtnlabs\/connectors\//,
```

Comment:
why you add this? just mock the response

Reply:
Oh, got you. I remove that.